components/actionbutton/stories/actionbutton.stories.js

```diff
@@ -116,6 +116,8 @@ export default {
 			handles: ["click .spectrum-ActionButton:not([disabled])"],
 		},
 		componentVersion: version,
+		componentPkgName: name,
+		componentGuidelinesName: "action-button",
```

Comment:
There's nothing preventing us from putting this into the `package.json` so I'd be open to that if it makes more sense to have it there. We really just need this string because the Spectrum guidelines site's URLs are consistent. I don't know how predictable the naming is otherwise, though.